### ISSUE TITLE
[BUG] Bloxstrap wont run.

### ISSUE THREAD

**Issue by wendingos** [pizzaboxer/bloxstrap#1588]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Whenever i launch roblox it doesnt run, and i noticed when i go to my tray, when i launch it and let it load a little a caution triangle comes up for a split second than goes away and closes bloxstrap.

**Comment by pizzaboxer** [comment]
Are you able to at least open the Bloxstrap Menu?

**Comment by wendingos** [comment]
I can open the bloxstrap menu, but it will not launch with my game/roblox

**Comment by pizzaboxer** [comment]
Try and launch Roblox again (through the Play Roblox shortcut), and afterwards, get Bloxstrap's latest log file from the `%localappdata%\Bloxstrap\Logs` folder and send it here please.

**Comment by wendingos** [comment]
Thanks so much for helping me, sorry that I didn't send it earlier.

Here is the log after launching.

**Comment by pizzaboxer** [comment]
Where's the log?

**Comment by wendingos** [comment]
It’s at the bottom of the page

**Comment by wendingos** [comment]
[Bloxstrap_20240411T000419Z (1).log](https://github.com/pizzaboxer/bloxstrap/files/14950566/Bloxstrap_20240411T000419Z.1.log)

**Comment by wendingos** [comment]
Please let me know if there is a fix for this.

**Comment by pizzaboxer** [comment]
Oh, I misunderstood your issue, my bad.

Does Roblox work if you use the original launcher? https://github.com/pizzaboxer/bloxstrap/wiki/Switching-between-Roblox-and-Bloxstrap

**Comment by wendingos** [comment]
No it does not, but with bloxstrap uninstalled it does.
I reinstalled twice. Sorry if I miss understand your question

**Comment by pizzaboxer** [comment]
Roblox works only when Bloxstrap isn't installed? You're sure of that?

**Comment by wendingos** [comment]
I tried once more, and it does

**Comment by wendingos** [comment]
Any fix?

**Comment by pizzaboxer** [comment]
Not really, because it doesn't really make any sense as to why that would be happening. I don't think I've seen an issue like this before, because the original launcher should be working at the very least.

Just to clarify, you were using the "Roblox Player" shortcut, right?
